### DISCUSSION TITLE
New release

### DISCUSSION THREAD

**Discussion by kingosticks** [librespot-org/librespot#1203] [General]
The dev branch is not in a position to be released and I don't see that changing anytime soon. I know we didn't want to do any further 0.4.x releases but there are good improvements available we could backport in the meantime. 

I think others have been maintaining private forks incorporating some of these changes. There's of course nothing wrong with this but I think it's healthier for the project in the long-term if we try and keep things together. 

I'm proposing we only do trivial backports. For anything more complicated, let's cross that bridge if we come to it. Hopefully there's a way we can do this without any impact to dev work that might be going on in parallel.

**Comment by herrernst** [comment]
Thanks for your status update. What are the blockers for releasing? For me, `dev` works really good, except one blocker: #1179. I could try assist debugging this maybe with some help from you, @kingosticks

I also think for the health of the project it's necessary to have regular releases and not scattered forks with some features applied.

In addition to backporting trivial features, I suggest these things:
• mark release blockers with a label in issues
• maybe at least release a Alpha/Beta, i.e. `0.5-alpha1` or so (that we have at least a git tag)
